playwright/src/test/java/com/microsoft/playwright/TestPageBaseClass.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.microsoft.playwright;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import com.microsoft.playwright.Browser.NewContextOptions;
+import com.microsoft.playwright.junit.PageTest;
+import static com.microsoft.playwright.Utils.getBrowserNameFromEnv;
+
+import static com.microsoft.playwright.assertions.PlaywrightAssertions.assertThat;
+import static com.microsoft.playwright.options.ColorScheme.DARK;
+
+class TestPageBaseClass extends PageTest {
```

Comment:
We should be able to switch our own test harness to the new API and get a better coverage I think.

Reply:
We should probably do it seperately, to avoid a gigantic Pull Request ?